Fix resizing of team page so profiles wrap nicely

wendi-yu opened the pull request:
Changed bootstrap grid size to allow the number of profiles per row
to scale nicely based on screen width, instead of immediately changing
to only one per row - replaced md columns with sm ones

Commit:
Fix resizing of team page so profiles wrap nicely

Changed bootstrap grid size to allow the number of profiles per row
to scale nicely based on screen width, instead of immediately changing
to only one per row - replaced md columns with sm ones

jacobdeery reviewed (commented):
Very nice! Looks much better on mobile.

The only regression I'm seeing on desktop is when zooming way out or way in, the images will start to collide with the text:
![image](https://user-images.githubusercontent.com/13474696/74608186-0617aa80-50ad-11ea-858d-528955c09d5f.png)
![image](https://user-images.githubusercontent.com/13474696/74608192-13cd3000-50ad-11ea-80ad-9fa24cab2230.png)

Commit:
Fix resizing for large screens

Also reorganized bootstrap layout and added "software" to list of
engineerings :)

wendi-yu commented:
today i discovered chrome's developer tools I think I've really grown a lot as a person :)

jacobdeery reviewed (approved):
👌
